tests/network/vmi_networking.go

```diff
@@ -506,6 +506,30 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		})
 	})
 
+	It("VMI with an interface that has ACPI Index set", func() {
+		const acpiIndex = 101
+		const pciAddress = "0000:01:00.0"
+		iface := *v1.DefaultBridgeNetworkInterface()
+		iface.ACPIIndex = acpiIndex
+		iface.PciAddress = pciAddress
+		testVMI := libvmi.NewAlpine(
+			libvmi.WithInterface(iface),
+			libvmi.WithNetwork(v1.DefaultPodNetwork()),
+		)
+		testVMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), testVMI)
+		Expect(err).ToNot(HaveOccurred())
+
+		libwait.WaitUntilVMIReady(testVMI, console.LoginToAlpine)
+
+		err := console.SafeExpectBatch(testVMI, []expect.Batcher{
+			&expect.BSnd{S: "\n"},
+			&expect.BExp{R: console.PromptExpression},
+			&expect.BSnd{S: "cat /sys/bus/pci/devices/" + pciAddress + "/acpi_index\n"},
+			&expect.BExp{R: strconv.Itoa(acpiIndex)},
```

Comment:
Why don't you check the name of the interface is `eno101`?

Reply:
The VM image is set to use the `ethN` naming scheme, like all "standard" images.
To change this, there is a need to configure the boot command line and tweak the udev settings, then reboot. I do not think we need it to assure it is seen in the guest.